test/e2e/operator/deployment_api.go

```diff
@@ -81,7 +81,7 @@ var _ = deploy.DescribeForSome("API", func(d *deploy.Deployment) bool {
 		k8sver = *ver
 
 		// All tests are expected to complete in 5 minutes.
-		// We need to set up the global variables indirectly to avoid a watning about cancel not being called.
+		// We need to set up the global variables indirectly to avoid a waiting about cancel not being called.
```

Comment:
"avoid a *warning* about"

This should go into that separate spelling PR.

Reply:
Done.

Reply:
The new PR for fixing typos is here: https://github.com/intel/pmem-csi/pull/702